File: Samples/PurchasePath/Source/Objective-C/PurchasePathExtension/CustomCard/NightOwlCardView.m
#import "NightOwlCardView.h"

@implementation NightOwlCardView

+ (instancetype)nightOwlCardView {
    return (NightOwlCardView *)[[[NSBundle mainBundle] loadNibNamed:NSStringFromClass(self) owner:self options:nil] firstObject];
}

@end

File: samples/PurchasePath/Source/PurchasePathExtension/Custom Card/NightOwlCardView.swift
import UIKit

class NightOwlCardView: UIView {

    @IBOutlet weak var titleLabel: UILabel!
    @IBOutlet weak var bodyLabel: UILabel!
    @IBOutlet weak var actionButton: UIButton!
    @IBOutlet weak var imageView: UIImageView!

    class func nightOwlCardView() -> NightOwlCardView {
        return (Bundle.main.loadNibNamed("NightOwlCardView", owner: self, options: nil)?.first as? NightOwlCardView)!
    }

}
